test/integration/expose-keptn-metric/no-metrics.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: test
+  name: $NAME-job1
+  namespace: $NAMESPACE
 spec:
-  backoffLimit: 0
+  backoffLimit: 3
```

Comment:
why this? This was a safety net that when we have a slow env, we let the job try for a few times until it actually fails